--- Logging error ---
Traceback (most recent call last):
  File "/home/yl254115/miniconda3/lib/python3.7/logging/__init__.py", line 1029, in emit
    self.flush()
  File "/home/yl254115/miniconda3/lib/python3.7/logging/__init__.py", line 1009, in flush
    self.stream.flush()
OSError: [Errno 28] No space left on device
Call stack:
  File "/neurospin/unicog/protocols/intracranial/Syntax_with_Fried/Code/Main/spikes/generate_multichannel_spike_epochs.py", line 86, in <module>
    epochs_spikes, epochsTFR_spikes = analyses_single_unit.generate_epochs_spikes(ch, channel_name, events_spikes, event_id, metadata, settings, params, preferences)
  File "../functions/analyses_single_unit.py", line 23, in generate_epochs_spikes
    baseline=None, preload=True, picks=None)
  File "</home/yl254115/miniconda3/lib/python3.7/site-packages/mne/externals/decorator.py:decorator-gen-180>", line 2, in __init__
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/utils/_logging.py", line 90, in wrapper
    return function(*args, **kwargs)
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/epochs.py", line 1960, in __init__
    event_repeated=event_repeated, verbose=verbose)
  File "</home/yl254115/miniconda3/lib/python3.7/site-packages/mne/externals/decorator.py:decorator-gen-171>", line 2, in __init__
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/utils/_logging.py", line 90, in wrapper
    return function(*args, **kwargs)
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/epochs.py", line 516, in __init__
    activate=activate)
  File "</home/yl254115/miniconda3/lib/python3.7/site-packages/mne/externals/decorator.py:decorator-gen-15>", line 2, in setup_proj
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/utils/_logging.py", line 90, in wrapper
    return function(*args, **kwargs)
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/io/proj.py", line 859, in setup_proj
    info['projs'] = activate_proj(info['projs'], copy=False)
  File "</home/yl254115/miniconda3/lib/python3.7/site-packages/mne/externals/decorator.py:decorator-gen-12>", line 2, in activate_proj
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/utils/_logging.py", line 90, in wrapper
    return function(*args, **kwargs)
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/io/proj.py", line 712, in activate_proj
    logger.info('%d projection items activated' % len(projs))
Message: '0 projection items activated'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/yl254115/miniconda3/lib/python3.7/logging/__init__.py", line 1029, in emit
    self.flush()
  File "/home/yl254115/miniconda3/lib/python3.7/logging/__init__.py", line 1009, in flush
    self.stream.flush()
OSError: [Errno 28] No space left on device
Call stack:
  File "/neurospin/unicog/protocols/intracranial/Syntax_with_Fried/Code/Main/spikes/generate_multichannel_spike_epochs.py", line 86, in <module>
    epochs_spikes, epochsTFR_spikes = analyses_single_unit.generate_epochs_spikes(ch, channel_name, events_spikes, event_id, metadata, settings, params, preferences)
  File "../functions/analyses_single_unit.py", line 23, in generate_epochs_spikes
    baseline=None, preload=True, picks=None)
  File "</home/yl254115/miniconda3/lib/python3.7/site-packages/mne/externals/decorator.py:decorator-gen-180>", line 2, in __init__
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/utils/_logging.py", line 90, in wrapper
    return function(*args, **kwargs)
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/epochs.py", line 1960, in __init__
    event_repeated=event_repeated, verbose=verbose)
  File "</home/yl254115/miniconda3/lib/python3.7/site-packages/mne/externals/decorator.py:decorator-gen-171>", line 2, in __init__
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/utils/_logging.py", line 90, in wrapper
    return function(*args, **kwargs)
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/epochs.py", line 520, in __init__
    self.load_data()  # this will do the projection
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/epochs.py", line 556, in load_data
    self._data = self._get_data()
  File "</home/yl254115/miniconda3/lib/python3.7/site-packages/mne/externals/decorator.py:decorator-gen-178>", line 2, in _get_data
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/utils/_logging.py", line 90, in wrapper
    return function(*args, **kwargs)
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/epochs.py", line 1282, in _get_data
    'points ...' % (n_events, len(self._raw_times)))
Message: 'Loading data for 7290 events and 6001 original time points ...'
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File "/home/yl254115/miniconda3/lib/python3.7/logging/__init__.py", line 1029, in emit
    self.flush()
  File "/home/yl254115/miniconda3/lib/python3.7/logging/__init__.py", line 1009, in flush
    self.stream.flush()
OSError: [Errno 28] No space left on device
Call stack:
  File "/neurospin/unicog/protocols/intracranial/Syntax_with_Fried/Code/Main/spikes/generate_multichannel_spike_epochs.py", line 86, in <module>
    epochs_spikes, epochsTFR_spikes = analyses_single_unit.generate_epochs_spikes(ch, channel_name, events_spikes, event_id, metadata, settings, params, preferences)
  File "../functions/analyses_single_unit.py", line 23, in generate_epochs_spikes
    baseline=None, preload=True, picks=None)
  File "</home/yl254115/miniconda3/lib/python3.7/site-packages/mne/externals/decorator.py:decorator-gen-180>", line 2, in __init__
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/utils/_logging.py", line 90, in wrapper
    return function(*args, **kwargs)
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/epochs.py", line 1960, in __init__
    event_repeated=event_repeated, verbose=verbose)
  File "</home/yl254115/miniconda3/lib/python3.7/site-packages/mne/externals/decorator.py:decorator-gen-171>", line 2, in __init__
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/utils/_logging.py", line 90, in wrapper
    return function(*args, **kwargs)
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/epochs.py", line 520, in __init__
    self.load_data()  # this will do the projection
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/epochs.py", line 556, in load_data
    self._data = self._get_data()
  File "</home/yl254115/miniconda3/lib/python3.7/site-packages/mne/externals/decorator.py:decorator-gen-178>", line 2, in _get_data
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/utils/_logging.py", line 90, in wrapper
    return function(*args, **kwargs)
  File "/home/yl254115/miniconda3/lib/python3.7/site-packages/mne/epochs.py", line 1343, in _get_data
    logger.info("%d bad epochs dropped" % (n_events - len(good_idx)))
Message: '171 bad epochs dropped'
Arguments: ()
/neurospin/unicog/protocols/intracranial/Syntax_with_Fried/Code/Main/spikes/generate_multichannel_spike_epochs.py:90: DeprecationWarning: overwrite defaults to True in 0.18 but will change to False in 0.19, set it explicitly to avoid this warning
  epochs_spikes.save(os.path.join(path2epochs, filename_epochs))
